consensus/spos/bls/export_test.go

```diff
@@ -366,3 +367,13 @@ func (sr *subroundStartRound) ChangeEpoch(epoch uint32) {
 func (sr *subroundStartRound) IndexRoundIfNeeded(pubKeys []string) {
 	sr.indexRoundIfNeeded(pubKeys)
 }
+
+// SendSignature calls the unexported sendSignature function
```

Comment:
```suggestion
// SendSignatureForManagedKey calls the unexported sendSignatureForManagedKey function
```
should update comment as well